src/main.jsx

```diff
@@ -1,13 +1,32 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
-import { TodosProvider } from "./store/todos-context";
-import App from './App'
-import './index.css'
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import { TodosProvider } from './store/todos-context';
+// import App from './App'
+import './index.css';
+import SignUp from './components/SignUp';
+import Login from './components/LogIn';
+import ForgotPassword from './components/ForgotPassword';
 
+const router = createBrowserRouter([
+    {
+        path: '/',
+        element: <SignUp />,
+    },
+    {
+        path: 'login',
+        element: <Login />,
+    },
+    {
+        path: 'forgot-password',
+        element: <ForgotPassword />,
+    },
+]);
 ReactDOM.createRoot(document.getElementById('root')).render(
-  <React.StrictMode>
-    <TodosProvider>
-      <App />
-    </TodosProvider>
-  </React.StrictMode>,
-)
+    <React.StrictMode>
+        <TodosProvider>
+            {/* <App /> */}
```

Comment:
I reckon we could add a route (/dashboard) for `App` as well.  What do you think?